flowsa/flowbyfunctions.py

```diff
@@ -1230,3 +1312,30 @@ def aggregate_and_subset_for_target_sectors(df, method):
     df_subset = subset_df_by_sector_list(df_agg, sector_list)
 
     return df_subset
+
+
+def add_column_of_allocation_sources(df, attr):
```

Comment:
I recall discussing that we only really need the primary allocation source and not the helper source in this column

Reply:
Updated function to only include primary attribution source